src/main/java/com/neuvector/Scanner.java

```diff
@@ -322,6 +322,14 @@ private static String getMountPath(NVScanner nvScanner){
         return mountPath;
     }
 
+    private static String appendSELinuxSuffixIfRequired(final Boolean isSELinuxSuffixRequired, final String mountPath) {
```

Comment:
Method name (and the argument name) needs update.

Reply:
Commit: [a55ee47](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/a55ee4758cc3ed7dbc9abf39326de51df7a62917)